### ISSUE TITLE
Use macro string ids instead of int

### ISSUE THREAD

**Issue by dzervas** [DangerousPrototypes/BusPirate5-firmware#21]
It'd be nice, backwards compatible and from what I gathered from the code easy to use string ids instead of integers.

instead of `macro 1` we could do `macro get_status`

thoughts?

**Comment by DangerousPrototypes** [comment]
Yeah, I don't see why not. It would be good to get the thoughts of the macro mode developer:

https://forum.buspirate.com/t/ideas-for-executing-bus-syntax-macros-from-file/345/10

**Comment by henrygab** [comment]
The forum [post](https://forum.buspirate.com/t/ideas-for-executing-bus-syntax-macros-from-file/345/17) indicating this was fixed.  Was fixed by PR #24.
